docusaurus/docs/iOS/uikit/guides/working-with-attachments.md

```diff
@@ -270,7 +270,30 @@ Components.default.attachmentViewCatalog = MyAttachmentViewCatalog.self
 
 The `AttachmentViewCatalog` class is used to pick the `AttachmentViewInjector` for a message, in our case we only need to check if the message contains a workout attachment.
 
-If needed you can also send a message with a workout attachment directly from Swift just to test that everything works correctly:
+In case you want your attachments to be rendered along with other types of attachments, you can do this by adhering your custom injector to the `MixedAttachmentViewInjector`. For this, first, you need to register your custom injector to the `MixedAttachmentViewInjector` registry like so:
```

Comment:
maybe rephrase this: "by adhering your custom injector to the `MixedAttachmentViewInjector`". When I read it, I get the impression that I should implement a protocol, which is not the case.

Reply:
I see, makes sense, I'll simplify it 👍

Reply:
Done ✅